source/index.html.md

```diff
@@ -130,6 +130,7 @@ error categories, the last two digits define specific errors.
 * 0209: Two Factor authentication required
 * 0210: Device authentication required
 * 0211: Device authentication failed (Invalid IP or expired Token)
+* 0212: Needs special permissions for this request
```

Comment:
Maybe liaise with Aline to rephrase this one - not a fan of it.
Something like `Unauthorized access` or simply return a 404 or an existing code such as `1005: Invalid operation`
The other PR is fine but we may need to change the error code